dspace/config/config-definition.xml

```diff
@@ -32,17 +32,17 @@
         <!-- Allow user to override any configs in a local.cfg -->
         <!-- Any properties in this config will override defaults in dspace.cfg (or any included *.cfg file) -->
         <properties fileName="local.cfg" throwExceptionOnMissing="false" config-name="local" config-optional="true">
-            <!-- Reload this file if it has changed, and at least 1 minute (60,000 ms) has passed -->
-            <reloadingStrategy refreshDelay="60000"
+            <!-- Reload this file if it has changed, and at least 1 minute (5,000 ms) has passed -->
```

Comment:
Silly change, but these comments both still say "at least 1 minute".  They both need updating to say "at least 5 seconds (5,000ms) have passed".  After that comment change, I'm +1.